engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/openapi/StandaloneOpenApiGenerator.scala

```diff
@@ -0,0 +1,76 @@
+package pl.touk.nussknacker.engine.standalone.openapi
+
+import io.circe.Json
+import io.circe.syntax._
+import io.circe.generic.auto._
+import pl.touk.nussknacker.engine.standalone.StandaloneProcessInterpreter
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+object StandaloneOpenApiGenerator {
+
+  private val OPEN_API_VERSION = "3.0.0"
+
+  private val jsonEncoder = BestEffortJsonEncoder(failOnUnkown = true, getClass.getClassLoader)
+
+  private[standalone] def generateScenarioDefinition(processName: String,
+                                                     requestDefinition: Json,
+                                                     responseDefinition: Json,
+                                                     description: Map[String, String],
+                                                     tags: List[String]
+                                                    ): Json = {
+   val postOpenApiDefinition = generatePostOApiDefinition(
+      tags,
+      processName,
+      preetyPrintMap(description),
+      requestDefinition,
+      responseDefinition
+    )
+    jsonEncoder.encode(postOpenApiDefinition)
+  }
+
+  def generateOpenApi(pathWithInterpreter: List[(String, StandaloneProcessInterpreter)], oApiInfo: OApiInfo, serverDescription: OApiServer): String = {
```

Comment:
1. Can u split this method into one that returns documentation and the second one that render it as a json? In case if someone would like to change this definition before returning it or would like to verify sth in tests
2. Do you have plans to use it in `ProcessRoute` in some next PR or maybe at least let's write a TODO that we should use it there to make this feature available without need of changing docker image of this standalone-app

Reply:
1. Good idea, please find updated class
2. I didn't planned to do it in this PR - though I have added comment as you suggested